VitDeck/Assets/VitDeck/Validator/Rules/Sample/SampleRuleSet.cs

```diff
@@ -15,5 +15,7 @@ public override string RuleSetName
         public IRule rule2 = new SampleRule("ルール２", "カスタム設定値２");
         [Validation]
         public IRule unityVersionRule = new UnityVersionRule("Unityバージョンルール", "2017.4.15f1");
+        [Validation]
+        public IRule assetPathLengthRule = new AssetPathLengthRule("[A03]アセットパス長制限ルール");
```

Comment:
```suggestion
        public IRule assetPathLengthRule = new AssetPathLengthRule("[A03]アセットパス長制限ルール", 184);
```
サンプルにデフォルト値以外を指定してもらえると使い方が分かりやすいのでありがたいです。
お手数ですがテストだけ1パスReRunお願いします。